library/cache-library/hazelcast-cache-service/src/test/resources/hazelcast.xml

```diff
@@ -1,9 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <hazelcast xmlns="http://www.hazelcast.com/schema/config">
-    <group>
-        <name>test</name>
-        <password>password</password>
-    </group>
+    <cluster-name>test</cluster-name>
```

Comment:
Is there still the option to have a user name/password?

Reply:
[According to this migration guide there is not](https://docs.hazelcast.com/imdg/4.2/migration-guides#changes-in-security-configurations), it appears to have been deprecated for a while.